requirements-dev.txt

```diff
@@ -1,14 +1,7 @@
-# test dependencies
```

Comment:
Updated a few version #'s and changed the contents of the file to be just an alphabetized list so it's easier to compare with the "pip list" output.